src/models/event.js

```diff
@@ -299,7 +310,16 @@ utils.extend(module.exports.MatrixEvent.prototype, {
     },
 
     /**
-     * Attempt to decrypt this event.
+     * Check if this event is currently being decrypted.
+     *
+     * @return {boolean} True if this event is currently being decrypted, else false.
+     */
+    isBeingDecrypted: function() {
+        return this._decrypting;
+    },
+
+    /**
+     * Start the process of trying to decrypt this event.
```

Comment:
mostly because the places this is called from don't actually care about when it completes. I could make it return a promise if you want.